tools/iptb-plugins/filecoin/local/localfilecoin.go

```diff
@@ -47,6 +47,9 @@ const defaultSectorsPath = "sectors"
 // defaultLogLevel is the value that will be used for GO_FILECOIN_LOG_LEVEL
 const defaultLogLevel = "3"
 
+// defaultRustLogLevel is the value that will be used for RUST_LOG
+const defaultRustLogLevel = "info"
```

Comment:
This changes seems unrelated, and also likely to add lots of noise back to our logs. Was it just for debugging?

Reply:
It is for debugging, yes. We've modified the default such that proofs logging is disabled by default.